tests/tests/fv3_ccpp_satmedmfq

```diff
@@ -0,0 +1,70 @@
+###############################################################################
+#
+#  FV3 CCPP satmedmfq test
```

Comment:
what is base test?

Reply:
fv3_satmedmf / fv3_ccpp_satmedmf (i.e. the existing satmedmf test that uses the other/older version of the scheme)